spec/controllers/deferred/confirmations_controller_spec.rb

```diff
@@ -1,6 +1,6 @@
-require 'spec_helper'
+require 'rails_helper'
```

Comment:
Looooong spec.  As a fan of Sani Metz, I'd like to see it broken out into smaller chunks.